### DISCUSSION TITLE
Reference Circuits

### DISCUSSION THREAD

**Discussion by mattjoyce** [shanteacontrols/OpenDeck#128] [Q&A] [answered]
Are there any reference circuits for folk to use to design variant boards?

**Comment by paradajz** [comment, marked as answer]
Yes.
https://github.com/shanteacontrols/OpenDeck/tree/master/bin/sch

**Reply by mattjoyce** [reply]
Thank you, these are a great learning resource.
I did search the wiki, perhaps a link on [Creating-custom-board-variant](https://github.com/shanteacontrols/OpenDeck/wiki/Creating-custom-board-variant)?

**Reply by paradajz** [reply]
Added under [Examples](https://github.com/shanteacontrols/OpenDeck/wiki/Creating-custom-board-variant#examples) section. :)
